docker/createml.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/opencontainers/go-digest"
 	"github.com/sirupsen/logrus"
 
-	"github.com/estesp/manifest-tool/types"
+	"manifest-tool/types"
```

Comment:
this should not be re-writing the imports; the `github.com/estesp` prefix should remain

Reply:
The types seams to be local so why reference them externally, which could cause some conflicts if you change them.

Reply:
Normally, you'd have `go.mod` start with the full remote path, then Go will know these are local and reference them appropriately, but also allow others to reference them properly externally (which won't work with them like this), so the `go.mod` should start with:

```go.mod
module github.com/estesp/manifest-tool
```